src/storage/src/hummock/sstable/block.rs

```diff
@@ -149,6 +166,8 @@ pub struct Block {
 
     /// Restart points.
     restart_points: Vec<RestartPoint>,
+
+    epoch_dictionary: EpochDictionary,
```

Comment:
need more documentation on what the structure of this dictionary is and how we use it.